doc/doxygen/headers/reordering.h

```diff
@@ -158,6 +158,62 @@
  *    transform the cell-local vertices so that they match the first cell's
  *    vertex ordering: i.e., the inverse orientation.
  *
+ * <h2>Orientations as Permutations of Vertices</h2>
+ * As discussed above, TriaAccessor::combined_face_orientation() returns a value
+ * less than ReferenceCell::n_face_orientations() which describes how the
+ * bounding object (i.e., a face) should be rotated to match the cell-local
+ * definition of that object. The encoding of triangles is
+ *
+ * <div class="threecolumn" style="width: 600px">
+ *   <div class="parent">
+ *     <div class="img" align="center">
+ *       @image html triangle-0.svg
+ *       @image html triangle-1.svg
+ *     </div>
+ *   </div>
```

Comment:
Do you think you could add some text that describes what the three numbers on each edge represent?

Reply:
Yes - done!